src/Parser/Annotation.php

```diff
@@ -66,7 +66,7 @@ abstract class Annotation implements Arrayable
      * @param null|string $method
      * @param null|Version $version
      */
-    public function __construct(
+    final public function __construct(
```

Comment:
To fix following Psalm error:

```
ERROR: UnsafeInstantiation - tests/Parser/Annotations/AnnotationTest.php:32:18 - Cannot safely instantiate class Mill\Parser\Annotation with "new $class_name" as its constructor might change in child classes (see https://psalm.dev/229)
                (new $annotation($this->getApplication(), $content, __CLASS__, __METHOD__))->process();


```

Reply:
BTW, it can also use the `@psalm-consistent-constructor` annotation for the abstract `Annotation` class.

And it's up to the repository owner.